plan/physical_plan_test.go

```diff
@@ -335,16 +335,16 @@ func (s *testPlanSuite) TestCBO(c *C) {
 			best: "Index(t.c_d_e)[[<nil>,+inf]]",
 		},
 		{
-			sql:  "select * from t where (t.c > 0 and t.c < 1) or (t.c > 2 and t.c < 3) or (t.c > 4 and t.c < 5) or (t.c > 6 and t.c < 7) or (t.c > 9 and t.c < 10)",
-			best: "Index(t.c_d_e)[(0,1) (2,3) (4,5) (6,7) (9,10)]",
+			sql:  "select * from t where (t.c > 0 and t.c < 1)",
+			best: "Index(t.c_d_e)[(0,1)]",
 		},
 		{
-			sql:  "select sum(t.a) from t where t.c in (1,2) and t.d in (1,3) group by t.d order by t.d",
-			best: "Index(t.c_d_e)[[1 1,1 1] [1 3,1 3] [2 1,2 1] [2 3,2 3]]->HashAgg->Sort->Trim",
+			sql:  "select sum(t.a) from t where t.c in (1,2) and t.d in (1,3) and t.e in (1) group by t.d order by t.d",
```

Comment:
Basically the same as the former, add an `t.d` expression to force use the equal search.